website/docs/guides/managed_service_identity.html.markdown

```diff
@@ -71,12 +71,20 @@ Terraform can be configured to use managed identity for authentication in one of
 
 ### Configuring with environment variables
 
-Setting the `ARM_USE_MSI` environment variable to `true` tells Terraform to use a managed identity. In addition to a properly-configured management identity, Terraform needs to know the subscription ID and tenant ID to identify the full context for the Azure provider.
+Setting the `ARM_USE_MSI` environment variable to `true` tells Terraform to use a managed identity.
+
+By default, Terraform will use the system assigned identity for authentication. If users want to use user assigned identity instead, the `ARM_CLIENT_ID` have to be specified to the [client id](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/user_assigned_identity#client_id) of the identity.
```

Comment:
```suggestion
By default, Terraform will use the system assigned identity for authentication. To use a user assigned identity instead, you will need to specify the  [`client_id`](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/user_assigned_identity#client_id) of the identity (which can also be sourced from the Environment Variable `ARM_CLIENT_ID`).
```

Reply:
The link of "client id" here redirects to the document of `azurerm_user_assigned_identity.client_id` actually. Although I added the provider block argument alternatives after each env var, I still want to keep above link there to guide users to use the right attribute if they provision user assigned identity via Terraform.